iocage_lib/ioc_common.py

```diff
@@ -46,6 +46,11 @@
 from iocage_lib.dataset import Dataset
 
 INTERACTIVE = False
+# 4 is a magic number for default and doesn't refer
+# to the actual ruleset 4 in devfs.rules(!)
+IOCAGE_DEVFS_RULESET = 4
+# Leave lower devfs_ruleset ids to sysadmin
+MIN_DYNAMIC_DEVFS_RULESET = 1000
```

Comment:
I think this would be better as a property for iocage that way the user can configure this the way they do everything else.

Reply:
That would be a global configuration though. Where would you do that?

Reply:
In the same place as normal properties, and we would just set the default as 1000. Allowing each jail to have their own minimum floor.

Reply:
Done